### PR TITLE
Adding missing translations #828

### PR DESCRIPTION
**Pull request by JisanAR03**
fix - #828 
i make this PR again after target the correct file ,i'm very sry for my previous mistake

BLT use for translate language : i18n, i just make some changes on template i mean just rewrite some text with "{% trans '' %}" and also i did some changes on locale folder because i had translate some text manually because they are translate automatically .

Note : i run a command at bash is : **apt-get install gettext**  (_for install gettext pakage_)

I attach some image of before and after :
before:
       
image

after :

    
image

i fix all page like this for each language.

### CODE REVIEW

**Comment by AtmegaBuzz** [comment]
LGTM @DonnieBLT

File: website/templates/hunt.html [hunk 1]
```diff
@@ -224,15 +225,15 @@
                 </div>
                 <div class="form-group">
                     <select name="plan" value="Flea" id="bughunt-select-plan">
-                        <option value="Flea">Flea Plan $9.00 / Month</option>
-                        <option value="Ant">Ant Plan $99.00 / Month</option>
-                        <option value="Wasp">Wasp Plan $499.00 / Month</option>
-                        <option value="Scorpion">Scorpion Plan $1499.00 / Month</option>
+                        <option value="Flea">Flea Plan $9.00 / {% trans 'Month' %}</option>
```
Comment (DonnieBLT) [inline website/templates/hunt.html:228]: can add trans to these too

File: website/templates/index.html [hunk 2]
```diff
@@ -264,24 +264,24 @@ <h1 class="page-sub-header">Latest activity</h1>
...
+                <a href="/leaderboard/each-month" class="btn btn-default btn-block submit_button">{% trans "View Monthly" %}</a>
             </div>
         </div>
     </div>
 </div>
 
 <div class="row">
     <div class="col-lg-12">
-        <h1 class="page-sub-header">Bugheist is 100% free to use, Open Source and a non-commercial, not for profit initiative.</h1>
+        <h1 class="page-sub-header">Bugheist is 100% {% trans "free to use" %}, {% trans "Open Source and a non-commercials" %}, {% trans "not for profit initiative" %}.</h1>
```
Comment (DonnieBLT) [inline website/templates/index.html:284]: non-commercial

File: website/templates/mobile_about.html [hunk 3]
```diff
@@ -1,44 +1,43 @@
...
-<h2>Bugheist is 100% free to use, Open Source and a non-commercial, not for profit initiative. 
-    All prize money goes directly to the bug hunter.</h2>
-<p>Software code allows us to buy a gift for Mom or Dad on amazon.com in 7 seconds, watch our favorite
-    “House of Cards” episode on Netflix or read Yelp reviews about a new restaurant.</p>
-
-<p>When we can’t access the information we’re looking for on the internet within seconds, <i>we are not
-        happy.</i>
+{% load i18n %}
+<h1>{% trans 'About' %} {% env 'PROJECT_NAME' %}</h1>
+<h2>Bugheist is 100% {% trans 'free to use' %}, {% trans 'Open Source and a non-commercials' %}, {% trans 'not for profit initiative' %}.
```
Comment (DonnieBLT) [inline website/templates/mobile_about.html:4]:
```suggestion
<h2>{% env 'PROJECT_NAME' %} is 100% {% trans 'free to use' %}, {% trans 'Open Source and a non-commercials' %}, {% trans 'not for profit initiative' %}. 
```

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
a couple of tweaks

**Comment by JisanAR03** [comment]
@DonnieBLT  I will make PR again after update the code,, thanks for the review sir

File: website/templates/hunt.html (hunk 1 above)
Comment (DonnieBLT) [reply website/templates/hunt.html:229]: I mean like {% trans "Ant plan" %}

File: website/templates/hunt.html [hunk 4]
```diff
@@ -245,13 +247,13 @@
...
                 <div class="form-group" id="bughunt-submit">
-                    <button type="submit">START BUGHUNT!</button>
+                    <button type="submit">{% trans 'START BUGHUNT' %}!</button>
                 </div>
                 {% endif %}
         </div>
     </div>
 
-<h1 class="page-sub-header">Bugheist is 100% free to use, Open Source and a non-commercial, not for profit initiative.</h1>
+<h1 class="page-sub-header">Bugheist is 100% {% trans 'free to use' %}, {% trans 'Open Source and a non-commercial' %}, {% trans 'not for profit initiative' %}.</h1>
```
Comment (DonnieBLT) [inline website/templates/hunt.html:256]: Replace with project_name tag

**Review by DonnieBLT — CHANGES_REQUESTED** [review]

File: website/templates/hunt.html (hunk 4 above)
Comment (JisanAR03) [reply website/templates/hunt.html]: completed sir

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
also can translate the "Ant plan" etc

File: locale/de/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -18,244 +18,968 @@ msgstr ""
...
+"Um dies zu bestätigen, gehen Sie zu %(activate_url)s"
 
-#: website/templates/account/login.html:77
+#: website/templates/account/login.html:21
+#: website/templates/account/signup.html:17
+msgid ""
+"OWASP BLT is a bug logging tool to report issues and get points, companies "
+"are held accountable"
+msgstr ""
+"OWASP BLT ist ein Fehlerprotokollierungstool, um Probleme zu melden und "
```
Comment (DonnieBLT) [inline locale/de/LC_MESSAGES/django.po:138]: Let's have this be the project name so separate this out

File: website/templates/index.html [hunk 6]
```diff
@@ -264,24 +264,24 @@ <h1 class="page-sub-header">Latest activity</h1>
                         {% endif %}
                         <a href="/profile/{{ leader.username }}" class="text-capitalize leaderboard-name">{{leader.username|truncatechars:20 }}</a>
                         <span class="label label-success leaderboard-rank">{{ forloop.counter|ordinal }}</span>
-                        <div class="badge leaderboard-points">{{ leader.total_score }} Points</div>
+                        <div class="badge leaderboard-points">{{ leader.total_score }} {% trans "Points" %}</div>
                     </div>
                     {% endfor %}
                     {% else %}
-                    </br> Leaderboard reset for {% now "F" %}. Be first to find issues!
+                    </br> {% trans "Examples" %} {% now "F" %}. {% trans "Be first to find issues" %}!
```
Comment (DonnieBLT) [inline website/templates/index.html:271]: Check this - I think it needs to say leaderboard reset for

File: website/templates/index.html [hunk 7]
```diff
@@ -264,24 +264,24 @@ <h1 class="page-sub-header">Latest activity</h1>
...
+                <a href="/leaderboard/each-month" class="btn btn-default btn-block submit_button">{% trans "View Monthly" %}</a>
             </div>
         </div>
     </div>
 </div>
 
 <div class="row">
     <div class="col-lg-12">
-        <h1 class="page-sub-header">Bugheist is 100% free to use, Open Source and a non-commercial, not for profit initiative.</h1>
+        <h1 class="page-sub-header">Bugheist is 100% {% trans "free to use" %}, {% trans "Open Source and a non-commercial" %}, {% trans "not for profit initiative" %}.</h1>
```
Comment (DonnieBLT) [inline website/templates/index.html:284]: Change bugheist to the project name, variable

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
It's looking good just a few places where the project name is mentioned please make sure it's a variable and that it's not in a translation file

**Comment by JisanAR03** [comment]
> It's looking good just a few places where the project name is mentioned please make sure it's a variable and that it's not in a translation file

after solve those i made i more commit please check it out sir

File: website/templates/account/login.html [hunk 8]
```diff
@@ -18,18 +18,18 @@
           <a href="{% url 'index' %}" class="hover:no-underline text-white hover:text-white">{% env 'PROJECT_NAME' %}</a>
         </div>
         <p class="mt-6 font-normal text-center text-white text-xl md:mt-0">
-            OWASP BLT is a bug logging tool to report issues and get points, companies are held accountable.
+          OWASP BLT {% trans 'is a bug logging tool to report issues and get points, companies are held accountable' %}.
```
Comment (DonnieBLT) [inline website/templates/account/login.html:21]: here is another spot where OWASP BLT should be replaced with the {% PROJECT_NAME %} so in general - all areas where it says "Bugheist" or "OWASP BLT" should be  {% PROJECT_NAME %}

File: website/templates/account/signup.html [hunk 9]
```diff
@@ -14,18 +14,18 @@
           <a href="{% url 'index' %}" class="hover:no-underline text-white hover:text-white">{% env 'PROJECT_NAME' %}</a>
         </div>
         <p class="mt-6 font-normal text-center text-white text-xl md:mt-0">
-            OWASP BLT is a bug logging tool to report issues and get points, companies are held accountable.
+          OWASP BLT {% trans 'is a bug logging tool to report issues and get points, companies are held accountable' %}.
```
Comment (DonnieBLT) [inline website/templates/account/signup.html:17]: same here

File: website/templates/index.html [hunk 10]
```diff
@@ -264,24 +264,24 @@ <h1 class="page-sub-header">Latest activity</h1>
...
+                <a href="/leaderboard/each-month" class="btn btn-default btn-block submit_button">{% trans "View Monthly" %}</a>
             </div>
         </div>
     </div>
 </div>
 
 <div class="row">
     <div class="col-lg-12">
-        <h1 class="page-sub-header">Bugheist is 100% free to use, Open Source and a non-commercial, not for profit initiative.</h1>
+        <h1 class="page-sub-header">{% env 'PROJECT_NAME' %} is 100% {% trans "free to use" %}, {% trans "Open Source and a non-commercial" %}, {% trans "not for profit initiative" %}.</h1>
```
Comment (DonnieBLT) [inline website/templates/index.html:284]: This looks good, this is how it should be

File: website/templates/account/login.html (hunk 8 above)
Comment (DonnieBLT) [inline website/templates/account/login.html:21]:
```suggestion
          {% env 'PROJECT_NAME' %} {% trans 'is a bug logging tool to report issues and get points, companies are held accountable' %}.
```

File: website/templates/account/signup.html (hunk 9 above)
Comment (DonnieBLT) [inline website/templates/account/signup.html:17]:
```suggestion
          {% env 'PROJECT_NAME' %} {% trans 'is a bug logging tool to report issues and get points, companies are held accountable' %}.
```

**Comment by JisanAR03** [comment]
@DonnieBLT I'm sorry sir ,, i should notice that one,, should i commit one more commit sir?

**Comment by DonnieBLT** [comment]
I think {% load custom_tags %} has to be added to two of the templates

**Comment by JisanAR03** [comment]
> {% load custom_tags %}

"{% load custom_tags %}" is already exist sir on those two file 
ScreesShot:
image
image

**Comment by DonnieBLT** [comment]
yeah not sure why it's not passing - I think it may have something to do with it being in the accounts folder

**Comment by JisanAR03** [comment]
> yeah not sure why it's not passing - I think it may have something to do with it being in the accounts folder
sir when i check this on my local site it's showing error for .env file

**Comment by JisanAR03** [comment]
> yeah not sure why it's not passing - I think it may have something to do with it being in the accounts folder

sir i check this one again and it's work fine for signup and login page sir ..can you please check this out for one more time

**Comment by JisanAR03** [comment]
> yeah not sure why it's not passing - I think it may have something to do with it being in the accounts folder

@DonnieBLT  yes sir that was my mistake i didn't add "{% load custom_tags %}" now i wish everything works fine ..sorry for not understand the issue on first try .

**Review by DonnieBLT — APPROVED** [review]
Nice work!  Thank you for this useful contribution.

**Comment by JisanAR03** [comment]
@DonnieBLT  Thank you so much for you time sir

**Pull request merged**